src/components/cc-elasticsearch-info/cc-elasticsearch-info.js

```diff
@@ -68,29 +114,31 @@ export class CcElasticsearchInfo extends LitElement {
               <cc-icon size="lg" .icon=${iconInfo}></cc-icon>
               <span>${i18n('cc-elasticsearch-info.link.doc')}</span>
             `)}
-            ${elasticsearchLink != null ? html`
-              ${ccLink(elasticsearchLink.href, html`
-                <cc-img src="${ELASTICSEARCH_LOGO_URL}"></cc-img>
-                <span class="${classMap({ skeleton: (elasticsearchLink.href == null) })}">${i18n('cc-elasticsearch-info.link.elasticsearch')}</span>
-              `)}
-            ` : ''}
-            ${kibanaLink != null ? html`
-              ${ccLink(kibanaLink.href, html`
-                <cc-img src="${KIBANA_LOGO_URL}"></cc-img>
-                <span class="${classMap({ skeleton: (kibanaLink.href == null) })}">${i18n('cc-elasticsearch-info.link.kibana')}</span>
-              `)}
-            ` : ''}
-            ${apmLink != null ? html`
-              ${ccLink(apmLink.href, html`
-                <cc-img src="${APM_LOGO_URL}"></cc-img>
-                <span class="${classMap({ skeleton: (apmLink.href == null) })}">${i18n('cc-elasticsearch-info.link.apm')}</span>
-              `)}
-            ` : ''}
+
+            ${this._renderLinks(this.state)}
           </div>
       </cc-block>
     `;
   }
 
+  /**
+   * @param {ElasticSearchInfoStateLoaded|ElasticSearchInfoStateLoading} state
+   * @returns {TemplateResult[]}
+   * @private
+   */
+  _renderLinks (state) {
```

Comment:
nitpick: in other components we have a `skeleton` variable in the render and we pass it to the subrender. I'd like to do the same here with `_renderLinks (links, skeleton) {` and links would have `LinkLoaded|LinkLoading` as a type.